lib/heapsort.js

```diff
@@ -1,8 +1,25 @@
+// const MinHeap = require('./minheap.js')
+const { MinHeap } = require('./minheap');
 // This method uses a heap to sort an array.
-// Time Complexity:  ?
-// Space Complexity: ?
+// Time Complexity:  O(log n)
+// Space Complexity: O(n)
+
 function heapsort(list) {
```

Comment:
You're adding n elements to a heap (each add is log n) and so this sort is O(n log n) time complexity.